src/main/java/org/spongepowered/api/util/ban/Ban.java

```diff
@@ -83,21 +84,42 @@ static Ban of(org.spongepowered.api.entity.living.player.User user, Text.Literal
      *
      * @return The reason specified for the ban.
      */
-    Text.Literal getReason();
+    Text getReason();
 
     /**
-     * Gets the start date of the ban.
+     * Gets the creation date of the ban.
+     *
+     * <p>Note that this {@link Date} has no effect on whether or not a ban is
+     * active. Any ban for which {@link BanService#hasBan(Ban)} returns
+     * <code>true</code> will be used (when checking if a player can join,
+     * for example), regardless of its creation date.</p>
      *
      * @return Creation date of the ban
      */
-    Date getStartDate();
+    Date getCreationDate();
```

Comment:
This suggests that a ban starts as soon as it's created. What if a ban was set to be at some point in the future?